lib/ansible/modules/cloud/azure/azure_rm_managed_disk.py

```diff
@@ -174,7 +183,8 @@ def managed_disk_to_dict(managed_disk):
         disk_size_gb=managed_disk.disk_size_gb,
         os_type=managed_disk.os_type.lower() if managed_disk.os_type else None,
         storage_account_type=managed_disk.sku.name if managed_disk.sku else None,
-        managed_by=managed_disk.managed_by
+        managed_by=managed_disk.managed_by,
+        zones=managed_disk.zones if managed_disk.zones else None
```

Comment:
`zone=managed_disk.zones[0] if managed_disk.zones and len(managed_disk.zones) > 0 else ''`